src/shared/components/ProfilePage/Stats/DistributionGraph/index.scss

```diff
@@ -0,0 +1,93 @@
+@import "~styles/mixins";
+
+:global {
```

Comment:
Can't we avoid global styles? I guess, the same can be achieved with CSS modules.